WhereAmIAgain/ConfigurationWindow.cs

```diff
@@ -46,7 +46,11 @@ public override void Draw()
         
         ImGuiHelpers.ScaledDummy(10.0f);
 
-        ImGui.InputText("##InputString", ref Configuration.FormatString, 35);
+        if (ImGui.InputText("##InputString", ref Configuration.FormatString, 35))

```

Comment:
The 35 in this line limits the character limit of the formatting string to 35 characters.

Most people will either use the default, remove a segment or two of the string, or will only use different separators between the placenames. Some people might want the format string to be something like:
Region: {0} -- Territory: {1} -- Area: {2} -- SubArea: {3}

The limit of 35 doesn't allow this.
I leave this to your discretion.